docs/articles/clustering/split-brain-resolver.md

```diff
@@ -0,0 +1,221 @@
+---
+uid: split-brain-resolver
+title: Split Brain Resolver
+---
+# Split Brain Resolver
+
+When operating an Akka cluster you must consider how to handle [network partitions](https://en.wikipedia.org/wiki/Network_partition) (a.k.a. split brain scenarios) and machine crashes (including .NET CLR/Core and hardware failures). This is crucial for correct behavior if you use Cluster Singleton or Cluster Sharding, especially together with Akka Persistence.
+
+> Note: while this feature is based on [Lightbend Reactive Platform Split Brain Resolver](https://doc.akka.io/docs/akka/rp-16s01p02/scala/split-brain-resolver.html) feature description, its implementation is result of free contribution and interpretation of Akka.NET team. Lightbend team doesn't take any responsibility for the state and correctness of it.
```

Comment:
IMHO, no need to even mention this. You had to write all of the source code from scratch yourself.

Reply:
Ofcourse he did. However considering akka.net is being positioned as a port of the JVM version. It does not hurt to point out that this particular part could behave differently, by design.